content/en/opentelemetry/otel_collector_datadog_exporter.md

```diff
@@ -99,22 +112,131 @@ The above configuration enables the receiving of OTLP data from OpenTelemetry in
+
+#### Configure the filelog receiver
 
-- **Apply [Unified Service Tagging][8]**: Make sure you’ve configured your application with the appropriate resource attributes for unified service tagging. This ties Datadog telemetry together with tags for service name, deployment environment, and service version. The application should set these tags using the OpenTelemetry semantic conventions: `service.name`, `deployment.environment`, and `service.version`.
+Configure the filelog receiver using [operators][14]. For example, if there is a service `checkoutservice` that is writing logs to `/var/log/pods/services/checkout/0.log`, a sample log might look like this:
+
+```
+{"level":"info","message":"order confirmation email sent to \"jack@example.com\"","service":"checkoutservice","span_id":"197492ff2b4e1c65","timestamp":"2022-10-10T22:17:14.841359661Z","trace_id":"e12c408e028299900d48a9dd29b0dc4c"}
+```
+
+Example filelog configuration:
+
+```
+filelog:
+   include:
+     - /var/log/pods/**/*checkout*/*.log
+   start_at: end
+   poll_interval: 500ms
+   operators:
+     - id: parse_log
+       type: json_parser
+       parse_from: body
+     - id: trace
+       type: trace_parser
+       trace_id:
+         parse_from: attributes.trace_id
+       span_id:
+         parse_from: attributes.span_id
+   attributes:
+     ddtags: env:staging
+```
 
-### 4. Run the collector
+- `include`: The list of files the receiver tails 
+- `start_at: end`: Indicates to read new content that is being written 
+- `poll_internal`: Sets the poll frequency 
+- Operators:
+    - `json_parser`: Parses JSON logs. By default, the filelog receiver converts each log line into a log record, which is the `body` of the logs' [data model][15]. Then, the `json_parser` converts the JSON body into attributes in the data model.
+    - `trace_parser`: Extract the `trace_id` and `span_id` from the log to correlate logs and traces in Datadog. 
+
+#### Using Kubernetes
+
+There are multiple ways to deploy the OpenTelemetry Collector and Datadog Exporter in a Kubernetes infrastructure. For the filelog receiver to work, the [Agent/DaemonSet deployment][16] is the recommended deployment method.
+
+In containerized environments, applications write logs to `stdout` or `stderr`. Kubernetes collects the logs and writes them to a standard location. You need to mount the location on the host node into the Collector for the filelog receiver. Below is an [extension example][17] with the mounts required for sending logs. 
+
+```
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: otel-agent
+  labels:
+    app: opentelemetry
+    component: otel-collector
+spec:
+  template:
+    metadata:
+      labels:
+        app: opentelemetry
+        component: otel-collector
+    spec:
+      containers:
+        - name: collector
+          command:
+            - "/otelcol-contrib"
+            - "--config=/conf/otel-agent-config.yaml"
+          image: otel/opentelemetry-collector-contrib:0.61.0
```

Comment:
This is a bit old (latest version is 0.71.0). Not sure if we should deal with it on this PR (since this line was copied over) but we should try and update this to a more recent version

Reply:
Thanks for catching this, created a card for tracking: [DOCS-4918](https://datadoghq.atlassian.net/browse/DOCS-4918)